packages/office-ui-fabric-react/src/components/Button/Button.Props.ts

```diff
@@ -27,6 +27,12 @@ export interface IButtonProps extends React.AllHTMLAttributes<HTMLAnchorElement
   href?: string;
 
   /**
+   * Button uses primary color theme if defined.
```

Comment:
I would rather the documentation be simplified and say that it's a flag which changes the visual presentation of the button to be emphasized. The wording "use primary color theme" is both vague and too specific in some cases :)